docs/index.asciidoc

```diff
@@ -141,7 +142,15 @@ E.g: `/%{myfield}/`, `/test-%{myfield}/` are not valid paths
 If `append`, the file will be opened for appending and each new event will be written at the end of the file.
 If `overwrite`, the file will be truncated before writing and only the most recent event will appear in the file.
 
-[id="plugins-{type}s-{plugin}-common-options"]
```

Comment:
This line must remain in the document.  It pairs with Line 145 to pull in and render the "Common Options" section that is shared for all output plugins.  If it's not there, the documentation build will fail.